api/src/main/kotlin/edu/wgu/osmt/security/OAuthHelper.kt

```diff
@@ -13,12 +15,19 @@ import org.springframework.stereotype.Component
 @Component
 class OAuthHelper {
 
+    @Autowired
+    lateinit var appConfig: AppConfig
+
     fun readableUsername(user: OAuth2User?, default: String = UNAUTHENTICATED_USERNAME): String {
         return user?.name ?: default
     }
 
     fun readableUsername(jwt: Jwt?, default: String = UNAUTHENTICATED_USERNAME): String {
-        return jwt?.claims?.get("name") as String? ?: default
+        return jwt?.claims?.get(appConfig.userName) as String? ?: default
+    }
+
+    fun readableEmail(jwt: Jwt?, default: String = UNAUTHENTICATED_USERNAME): String {
```

Comment:
We can't prescribe that this is an email address. That's up to the JWT issuer. So can we please rename this as readableUserIdentifier?

Reply:
gotcha, changing it